apel-ssm.spec

```diff
@@ -100,6 +100,9 @@ rm -rf $RPM_BUILD_ROOT
 %doc %_defaultdocdir/%{name}
 
 %changelog
+* Tue Sep 19 2023 Nicholas Whyatt <nicholas.whyatt@stfc.ac.uk> - 3.4.0-1
+| - Removed python-ldap as a dependency
+
```

Comment:
The changelogs get updated once I'm on a release branch for a specific release, otherwise I'd forever be tweaking this section while feature branches are being merged.

Reply:
Removed: now squashing the two commits